rabbitmq/src/main/kotlin/io/qalipsis/plugins/rabbitmq/consumer/RabbitMqConsumerIterativeReader.kt

```diff
@@ -102,10 +110,36 @@ internal class RabbitMqConsumerIterativeReader(
         }
     }
 
-    private fun initMonitoringMetrics() {
+    private fun initMonitoringMetrics(scenarioName: ScenarioName, stepName: StepName) {
         meterRegistry?.apply {
-            meterBytesCounter = meterRegistry.counter("${meterPrefix}-bytes", meterTags!!)
-            meterRecordsCounter = meterRegistry.counter("${meterPrefix}-records", meterTags!!)
+            meterBytesCounter = meterRegistry.counter(scenarioName, stepName, "${meterPrefix}-bytes", eventTags!!)
```

Comment:
```suggestion
            meterBytesCounter = meterRegistry.counter(scenarioName, stepName, "${meterPrefix}-bytes", eventTags)
```

Here and everywhere.